wgpu-hal/src/gles/command.rs

```diff
@@ -1004,38 +1019,44 @@ impl crate::CommandEncoder<super::Api> for super::CommandEncoder {
         &mut self,
         start_vertex: u32,
         vertex_count: u32,
-        start_instance: u32,
+        base_instance: u32,
```

Comment:
Can we keep the name as it was before? Or rename both to `first_vertex`/`first_instance` everywhere?

Reply:
The naming across APIs looks to be a mess as well but we should at least try to keep it consistent with the WebGPU and Vulkan specs (both using `first_vertex`/`first_instance`).